packages/cashier/src/pages/deposit/crypto-deposit/crypto-deposit.tsx

```diff
@@ -3,32 +3,20 @@ import { Button, ButtonLink, Clipboard, Dropdown, Icon, Loading, Text } from '@d
 import { localize, Localize } from '@deriv/translations';
 import { CryptoConfig, getCurrencyName, isCryptocurrency, isMobile } from '@deriv/shared';
 import QRCode from 'qrcode.react';
-import { connect } from 'Stores/connect';
-import { TRootStore, TClientStore, TCryptoTransactionDetails } from 'Types';
+import { observer } from 'mobx-react-lite';
 import RecentTransaction from 'Components/recent-transaction';
+import { useStore } from '../../../hooks';
```

Comment:
@farzin-fs just a suggestion. maybe we could setup webpack to avoid these imports with many dots `"../../../../" `which we also have is spec.tsx files above, like we use `import { something } from 'Types'` ?

Reply:
@george-usynin-binary Yes this was already [suggested](https://github.com/binary-com/deriv-app/pull/6281#discussion_r990785865) by @hamid-deriv on #6281 and done there which is pending for you to review. To avoid double work, I will wait for that PR to merge first and then apply the changes here 🙇🏻

Reply:
@farzin-fs oh, cool. thanks a lot for the explanation